xla/backends/gpu/runtime/BUILD

```diff
@@ -403,6 +403,9 @@ xla_test(
         "//xla/stream_executor:stream_executor_h",
         "//xla/stream_executor:stream_executor_memory_allocator",
         "//xla/stream_executor/cuda:cuda_compute_capability",
+        "//xla/stream_executor/cuda:cudnn_plugin",
```

Comment:
You will need to move some under if_cuda_is_configured: https://github.com/openxla/xla/actions/runs/14654737681/job/41127713981?pr=25707